nav2_lifecycle_manager/src/lifecycle_manager.cpp

```diff
@@ -39,7 +39,7 @@ LifecycleManager::LifecycleManager()
   // The default set of node names for the nav2 stack
   std::vector<std::string> default_node_names{"map_server", "amcl",
     "planner_server", "controller_server",
-    "recoveries_server", "bt_navigator"};
+    "recoveries_server", "bt_navigator", "waypoint_follower"};
```

Comment:
Should `waypoint_follower` be a default?

Reply:
Its in the bringups, so yeah, it should be

Reply:
OK

Reply:
Oh, maybe I was a little too literal in reading that about this line in the default nodes.

Is your question whether this should be brought up at all by default? I think that's a legitimate discussion point. @orduno do you not want this to come up be default in the example launch files we provide?